<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateImageDialog</class>
 <widget class="QDialog" name="GenerateImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Image</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="XmlUi::GradientWidget" name="promptWidget" native="true">
     <layout class="QHBoxLayout" name="promptLayout" stretch="0,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="promptPixmap"/>
      </item>
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Image Resolution</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="spacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Width:</string>
           </property>
           <property name="buddy">
            <cstring>spinWidth</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinWidth">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>8000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Height:</string>
           </property>
           <property name="buddy">
            <cstring>spinHeight</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinHeight">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>8000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Advanced Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelDepth">
           <property name="text">
            <string>Calculation Depth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DoubleSlider" name="sliderDepth">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDetail">
           <property name="text">
            <string>Detail Level</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DoubleSlider" name="sliderDetail">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAntiAliasing">
           <property name="text">
            <string>Image Anti-Aliasing</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_2">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>51</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frameAntiAliasing">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <layout class="QHBoxLayout" name="_4">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioAANone">
                <property name="text">
                 <string>None</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioAALow">
                <property name="text">
                 <string>Low</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioAAMedium">
                <property name="text">
                 <string>Medium</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioAAHigh">
                <property name="text">
                 <string>High</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelMultiSampling">
           <property name="text">
            <string>Multi-Sampling</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frameMultiSampling">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioMSNone">
                <property name="text">
                 <string>None</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioMS2x2">
                <property name="text">
                 <string>2 x 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioMS4x4">
                <property name="text">
                 <string>4 x 4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioMS8x8">
                <property name="text">
                 <string>8 x 8</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="spacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XmlUi::GradientWidget</class>
   <extends>QWidget</extends>
   <header>xmlui/gradientwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>doubleslider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
  <tabstop>sliderDepth</tabstop>
  <tabstop>sliderDetail</tabstop>
  <tabstop>radioAANone</tabstop>
  <tabstop>radioAALow</tabstop>
  <tabstop>radioAAMedium</tabstop>
  <tabstop>radioAAHigh</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GenerateImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GenerateImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
